### PR TITLE
Feature/project overview page

### PR DESCRIPTION
**Pull request by emil-placheta**

### CODE REVIEW

File: packages/forms-web-app/src/pages/projects/project-information/view.njk [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	<div class="govuk-grid-row">
+		<div class="govuk-grid-column-one-third">
+			{% set activeId = "project-information" %}
+
+			{% include "projects/components/vertical-tabs-wrapper.njk" %}
+		</div>
+
+		<div class="govuk-grid-column-two-thirds">
+			{% include "projects/project-information/includes/about.njk" %}
+			<hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible"/>
```
[the remark this answers is not in the document]
Comment (emil-placheta) [reply packages/forms-web-app/src/pages/projects/project-information/view.njk:13]: gov uk classes component appear to be responsive, including this section break

File: packages/forms-web-app/src/pages/projects/project-information/utils/get-page-data.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const stripPrefixFromProposalType = (proposalType = '') => {
+	//regex: At the start of the string, find 2 word characters, followed by 2 number characters, followed by space, '-' and space
+	//Matches 'EN01 - '
+	const prefixPattern = /^\w{2}\d{2}\s-\s/;
```
[the remark this answers is not in the document]
Comment (emil-placheta) [reply packages/forms-web-app/src/pages/projects/project-information/utils/get-page-data.js:4]: yes, seen all prefixes

File: packages/forms-web-app/src/pages/projects/project-information/controller.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const logger = require('../../../lib/logger');
+const { getPageData } = require('./utils/get-page-data');
+
+const view = 'projects/project-information/view.njk';
+
+const getProjectOverview = (req, res, next) => {
+	const {
+		locals: { applicationData }
```
[the remark this answers is not in the document]
Comment (emil-placheta) [reply packages/forms-web-app/src/pages/projects/project-information/controller.js:8]: where does it come from?

[the remark this answers is not in the document]
Comment (jonathangoulding) [reply packages/forms-web-app/src/pages/projects/project-information/controller.js:8]: thats how the views pick up case ref etc without them needing to be set by controllers

**Review by jonathangoulding — APPROVED** [review]

**Pull request merged**